docs/tech/python-packages.md

```diff
@@ -0,0 +1,40 @@
+# Python packages
+
+A lot of our code is published as Python packages on [PyPI][].
+
+[PyPI]: https://pypi.org/
+
+## Development
+
+Make changes to the repository that contains the Python package using
+the normal GitHub flow.
+
+Bump the version in the repository. You might have to look in:
+
+- `setup.py`
+- `package/__init__.py`
+- `docs/conf.py`
+
+## Release
+
+Use [`git flow release`](../tools.html#git-flow) to publish a new release
+to GitHub. This will create a new tag which will cause Travis to build.
```

Comment:
I don't think I know enough about this to write the docs on it yet, but either somebody else can or I will do once I've figured out the details :)